apollo-api/src/commonTest/kotlin/com/apollographql/apollo/api/internal/SimpleResponseReaderTest.kt

```diff
@@ -1,453 +0,0 @@
-package com.apollographql.apollo.api.internal
-
-import com.apollographql.apollo.api.BigDecimal
-import com.apollographql.apollo.api.CustomScalarAdapter
-import com.apollographql.apollo.api.JsonElement
-import com.apollographql.apollo.api.EMPTY_OPERATION
-import com.apollographql.apollo.api.ResponseField
-import com.apollographql.apollo.api.CustomScalar
-import com.apollographql.apollo.api.CustomScalarAdapters
-import com.apollographql.apollo.api.internal.json.BufferedSourceJsonReader
-import com.apollographql.apollo.api.internal.json.JsonDataException
-import com.apollographql.apollo.api.internal.json.JsonUtf8Writer
-import com.apollographql.apollo.api.internal.json.Utils
-import okio.Buffer
-import kotlin.test.Test
-import kotlin.test.assertEquals
-import kotlin.test.assertTrue
-import kotlin.test.fail
-
-class SimpleResponseReaderTest {
```

Comment:
Is this test case not relevant any more?

Reply:
Once we handle cache on a separate path, I'm planning to simplify the `ResponseReader` API to be much closer to a `JsonReader` Api. It would then not work on `ResponseFields` anymore and these tests wouldn't be relevant anymore. We're not there yet though so I'll bring them back for now.